src/components/Auth/Logo/index.js

```diff
@@ -4,7 +4,7 @@ import logo from 'images/logo_white.png';
 
 const Logo = ({ label }) => {
   return (
-    <div className="row logo-auth">
+    <div className="logo-auth">
```

Comment:
This is inconsistent with `auth-alert` (auth is used as a prefix). So it should be `auth-logo` for consistency.

Reply:
Updated in https://github.com/rafayet-monon/surveyor/pull/13/commits/85d52aa97d04c895ad920aa3419db76cde1ba749